litestar/testing/helpers.py

```diff
@@ -378,16 +378,16 @@ def create_async_test_client(
         .. code-block:: python
 
             from litestar import get
-            from litestar.testing import create_test_client
+            from litestar.testing import create_async_test_client
 
 
             @get("/some-path")
             def my_handler() -> dict[str, str]:
                 return {"hello": "world"}
 
 
-            def test_my_handler() -> None:
-                with create_test_client(my_handler) as client:
+            async def test_my_handler() -> None:
+                async with create_async_test_client(my_handler) as client:
                     response == client.get("/some-path")
```

Comment:
```suggestion
                    response = await client.get("/some-path")
```

You probably have to remove the `==` in the sync version of the doc as well

https://github.com/litestar-org/litestar/blob/14759558cf231c61049fdd5b65a39eccb8841616/litestar/testing/helpers.py#L136

Reply:
> You probably have to remove the == in the sync version of the doc as well

@cbscsm can you make that change? 🙂

Reply:
Sorry, I missed that comment. I fixed it.